run-tests.sh

```diff
@@ -12,5 +12,5 @@ printf "**** Complete backend tests ***\n"
 # Test the frontend
 printf "**** ${BLUE}Begin frontend tests${NC} ***\n"
 docker-compose run \
-  --rm frontend npm test -- --watch
+  --rm frontend npm test -- --watchAll=false
```

Comment:
Does this make a difference, since you updated the travis configs with different settings?

Reply:
changed back in 82cb072